README.md

```diff
@@ -3,44 +3,15 @@
 
 Browser solidity is a browser based solidity compiler. To use either visit [https://chriseth.github.io/browser-solidity](https://chriseth.github.io/browser-solidity) or clone/download this repo and open `index.html` in your browser.
 
-#Nodejs usage
+#Load the extension
+Extensions that you download from the Chrome Web Store are packaged up as .crx files, which is great for distribution, but not so great for development. Recognizing this, Chrome gives you a quick way of loading up your working directory for testing. Let's do that now.
 
-To use the solidity compiler via nodejs you can install it via npm
+Visit chrome://extensions in your browser (or open up the Chrome menu by clicking the icon to the far right of the Omnibox:  The menu's icon is three horizontal bars. and select Extensions under the Tools menu to get to the same place).
 
-	npm install solc
+Ensure that the Developer mode checkbox in the top right-hand corner is checked.
 
-And then use it like so:
+Click Load unpacked extension… to pop up a file-selection dialog.
 
```

Comment:
If this was added to the README instead of replaced, then merging would be more likely i would have thought.